### PR TITLE
Revised `Purpose`.

### PR DESCRIPTION
**Pull request by m1cm1c**
As the chapter "Purpose" is nearing its final form, I revised it to fix grammar mistakes and to improve wording without changing the meaning of anything.

### CODE REVIEW

File: doc/Requirements Specification/Parts/Purpose.lyx [hunk 1]
```diff
@@ -474,7 +474,7 @@ key "KrogmannPhD"
 
 \end_inset
 
-, it aims to develop such a software adding dynamic properties to a 
+, it aims to develo a software adding dynamic properties to a
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Purpose.lyx]: Typo

File: doc/Requirements Specification/Parts/Purpose.lyx [hunk 2]
```diff
@@ -398,7 +398,7 @@ gls{SoMoX}
 \end_inset
 
 .
- It aims to conduct performance measurements on source code of a software,
+ It aims to conduct performance measurements on source code of a software
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Purpose.lyx]: “a software”! Maybe use “It aims to conduct performance measurements on software’s source code”?

File: doc/Requirements Specification/Parts/Purpose.lyx [hunk 3]
```diff
@@ -849,7 +850,7 @@ name "/B50/"
 
 \end_inset
 
- Beagle does no performance analysis or prediction.
+ Beagle does not do performance analysis nor prediction.
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Purpose.lyx]: I think this should be ...neither...nor...

**Comment by RomanLangrehr** [comment]
> As the chapter "Purpose" is nearing its final form

I think it would have been a good idea to wait for #166

File: doc/Requirements Specification/Parts/Purpose.lyx (hunk 2 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Purpose.lyx]: Ack.

File: doc/Requirements Specification/Parts/Purpose.lyx (hunk 3 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Purpose.lyx]: Ack.

**Comment by m1cm1c** [comment]
> I think it would have been a good idea to wait for #166

You're right. This is blocked until #166 is merged.

**Comment by RomanLangrehr** [comment]
> You're right. This is blocked until #166 is merged.

I don' think that is helpfull, as you already did the work.

**Comment by m1cm1c** [comment]
You only added a single block to `Purpose.lyx` and replaced a single line with it. I'll have to resolve a maximum of one conflict.

**Comment by m1cm1c** [comment]
I revised the new parts. One mistake isn't fixed but @RomanLangrehr will do this soon.

File: doc/Requirements Specification/Parts/Purpose.lyx [hunk 4]
```diff
@@ -474,7 +474,7 @@ key "KrogmannPhD"
 
 \end_inset
 
-, it aims to develop such a software adding dynamic properties to a 
+, it aims to develop a software adding dynamic properties to a
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Purpose.lyx]: a software

**Comment by m1cm1c** [comment]
I forgot to tell you that I changed /OP20/ substantially. Please check whether what I wrote is correct.

File: doc/Requirements Specification/Parts/Purpose.lyx (hunk 4 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Purpose.lyx]: Fixed it.

**Comment by viridispirus** [comment]
lgtm

**Comment by RomanLangrehr** [comment]
[PDF](http://m1cm1c.github.io/Beagle/branches/grammar-wording-purpose/Requirements%20Specification.pdf)

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**